zookeeper-docs/src/documentation/content/xdocs/zookeeperAdmin.xml

```diff
@@ -1641,6 +1641,24 @@ server.3=zoo3:2888:3888</programlisting>
                 Default is "10000".</para>
             </listitem>
           </varlistentry>
+
+          <varlistentry>
+            <term>networkBufferSize</term>
+
+            <listitem>
+              <para>(Java system property: zookeeper.<emphasis
+                      role="bold">networkBufferSize</emphasis>)</para>
+
+              <para><emphasis role="bold">New in 3.6.0:</emphasis>
+                Sets the socket send and receive buffer size between leader
```

Comment:
nit: buffer size in bytes

it probably should be obvious, but it's worth being extra clear.